src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1542,3 +1542,1398 @@
+          text: >
+            az webapp hybrid-connection remove -g [myResourceGroup] -n [myAppName]
+            --namespace [hybridConnectionNamespace] --hybrid-connection [hybridConnectionName]
+"""
+
+helps['appservice hybrid-connection'] = """
+    type: group
+    short-summary: commands that interact with hybrid connections on an app service level - existing command sets which key each app uses to connect to the hybric connection
+    examples:
+        - name: set the key the apps should use
+          text: >
+            az appservice hybrid-connection set-key --asp [hybridConnectionAppServicePlan] -g [appServicePlanResourceGroup]
+            --namespace [hybridConnectionNamespace] --hybrid-connection [hybridConnectionName]
+            --key-type ["primary" | "secondary"]
+"""
+
+helps['appservice hybrid-connection set-key'] = """
+    type: command
+    short-summary: specifies which key all the apps using a hybrid connection should use when connecting to it
+    parameters:
+        - name: --asp
+          type: string
+          short-summary: 'Appservice plan'
+        - name: --hybrid-connection
+          type: string
+          short-summary: 'Hybrid connection name'
+        - name: --namespace
+          type: string
+          short-summary: 'Hybrid connection namespace'
+        - name: --key-type
+          type: string
+          short-summary: 'Which key (primary or secondary) should be used'
+    examples:
+        - name: set the key the apps should use
+          text: >
+            az appservice hybrid-connection set-key --asp [hybridConnectionAppServicePlan] -g [appServicePlanResourceGroup]
+            --namespace [hybridConnectionNamespace] --hybrid-connection [hybridConnectionName]
+            --key-type ["primary" | "secondary"]
+"""
+
+helps['appservice vnet-integration'] = """
+    type: group
+    short-summary: commands that operate on virtual network integrations in a specified app service plan
+    examples:
+        - name: list the virtual network integrations
+          text: >
+            az appservice vnet-integration list -g [myResourceGroup] --asp [myAspName]
+"""
+
+helps['appservice vnet-integration list'] = """
+    type: command
+    short-summary: lists all the virtual networks in a specified app service plan
+    parameters:
+        - name: --asp
+          type: string
+          short-summary: 'Appservice plan'
+    examples:
+        - name: list the virtual network integrations
+          text: >
+            az appservice vnet-integration list -g [myResourceGroup] --asp [MyAspName]
+"""
+
+helps['webapp vnet-integration'] = """
+    type: group
+    short-summary: commands that list, add and remove a vnet-integration from an app
+    examples:
+        - name: list the virtual network integrations
+          text: >
+            az webapp vnet-integration list -g [myResourceGroup] -n [myAppName]
+        - name: add a swift virtual network to an app
```

Comment:
don't use Swift - this is an internal code name. Please check with Christina on the help text here.